Add missing documentation (#410)
powsybl/powsybl-dynawo: docs/dynamic_simulation/output-variables-mapping.md

> # Output variables mapping

Reviewer: ```suggestion
# Output variable mapping
```

> The output variables domain specific language allow a user to configure the curves or final state values Dynawo will export at the end of the simulation.

Reviewer: ```suggestion
The output variable domain specific language allows a user to configure the curves or final state values Dynawo will export at the end of the simulation.
```

This DSL defines the `curve` and `fsv`keywords.

> The `curve` keyword combined with the `variable` field create a single curve for a dynamic model. One identifies a dynamic model by its ID, the same as the one used in the [Dynamic Models DSL](dynamic-models-mapping). The variable to plot is identified by its name.

Reviewer: ```suggestion
The `curve` keyword combined with the `variable` field create a single curve for a dynamic model. The dynamic model is identified by its ID, the same as the one used in the [Dynamic Models DSL](dynamic-models-mapping). The variable to plot is identified by its name.
```
